WordPress/Classes/ViewRelated/People/InvitePersonViewController.swift

```diff
@@ -383,8 +391,14 @@ private extension InvitePersonViewController {
         messageTextView.backgroundColor = .listForeground
     }
 
+    func setupPlaceholderLabel() {
+        placeholderLabel.text = NSLocalizedString("Custom message...", comment: "Placeholder for Invite People message field.")
```

Comment:
nitpick / optional, use encoded ellipsis symbol instead of three dots 
```suggestion
        placeholderLabel.text = NSLocalizedString("Custom message…", comment: "Placeholder for Invite People message field.")
```

Reply:
Oh thanks! I meant to change that but obviously forgot.